types/node/index.d.ts

```diff
@@ -26,12 +26,18 @@
 //                 Lishude <https://github.com/islishude>
 //                 Andrew Makarov <https://github.com/r3nya>
 //                 Zane Hannan AU <https://github.com/ZaneHannanAU>
+//                 Thomas den Hollander <https://github.com/ThomasdenH>
 //                 Eugene Y. Q. Shen <https://github.com/eyqs>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /** inspector module types */
 /// <reference path="./inspector.d.ts" />
 
+// Forward-declare SharedArrayBuffer (available starting es2017)
+interface SharedArrayBuffer {
+    readonly byteLength: number;
```

Comment:
This somehow reduces the type safety as everything having a `readonly byteLength: number;` matches here. Should we add more properties?
I'm not sure here.

Reply:
The only other property is the slice method, I have added it.

Reply:
This change caused one of your tests to fail.

Reply:
Of course.